### PR TITLE
Ak random controllers

### LINKED ISSUE #14: make random controller for  merchants, customers, transactions

### PR DESCRIPTION
**Pull request by akintner**
adds random_controllers for merchant, customer, transaction

adds random_specs for each of the above models

all tests passing

closes #14

### CODE REVIEW

File: app/controllers/api/v1/transactions/random_controller.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class Api::V1::Transactions::RandomController < ApplicationController
+
+  def show
+    render json: Transaction.take
```
Comment (Laszlo-JFLMTCO) [inline app/controllers/api/v1/transactions/random_controller.rb:4]: @akintner `take` doesn't really provide a random object, but always returns the same.

Link to spec: http://apidock.com/rails/ActiveRecord/FinderMethods/take

`Transaction.all.shuffle.first` would return a different, random object each time it is run. I suggest we update this implementation, calling `all.shuffle.first` through a `random` method within the model.

File: app/controllers/api/v1/merchants/random_controller.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+class Api::V1::Merchants::RandomController < ApplicationController
+
+  def show
+    render json: Merchant.take
```
Comment (Laszlo-JFLMTCO) [inline app/controllers/api/v1/merchants/random_controller.rb:4]: @akintner `take` doesn't really provide a random object, but always returns the same.

Link to spec: http://apidock.com/rails/ActiveRecord/FinderMethods/take

`Merchant.all.shuffle.first` would return a different, random object each time it is run. I suggest we update this implementation, calling `all.shuffle.first` through a `random` method within the model.

File: app/controllers/api/v1/customers/random_controller.rb [hunk 3]
```diff
@@ -0,0 +1,6 @@
+class Api::V1::Customers::RandomController < ApplicationController
+
+  def show
+    render json: Customer.take
```
Comment (Laszlo-JFLMTCO) [inline app/controllers/api/v1/customers/random_controller.rb:4]: @akintner `take` doesn't really provide a random object, but always returns the same.

Link to spec: http://apidock.com/rails/ActiveRecord/FinderMethods/take

`Customer.all.shuffle.first` would return a different, random object each time it is run. I suggest we update this implementation, calling `all.shuffle.first` through a `random` method within the model.

**Comment by akintner** [comment]
fixed in a different branch, closing

**Pull request closed without merging**